File: backend/CateringAPI/Models/EventMenuItem.cs
using System.ComponentModel.DataAnnotations;

namespace CateringAPI.Models;

public class EventMenuItem
{
    public int Id { get; set; }

    public int EventId { get; set; }
    public Event? Event { get; set; }

    public int MenuItemId { get; set; }
    public MenuItem? MenuItem { get; set; }

    public int QtyOrdered { get; set; }
    public int QtyLeftover { get; set; }
}

File: frontend/src/types/EventMenuItem.ts
import type { MenuItem } from './MenuItem';
import type { Event } from './Event';

export type EventMenuItem = {
  id: number;
  eventId: number;
  event?: Event;
  menuItemId: number;
  menuItem?: MenuItem;
  qtyOrdered: number;
  qtyLeftover: number;
};
